src/libs/sfdk/virtualmachine.h

```diff
@@ -84,6 +84,7 @@ class SFDK_EXPORT VirtualMachine : public QObject
         ShrinkStorageSize = 0x08,
         OptionalHeadless = 0x10,
         Snapshots = 0x20,
+        SwapMemory = 0x40,
```

Comment:
I don't quite follow here. We have VirtualMachine::Features::SwapMemory, and we have VirtualMachineInfo::swapSupported. They are almost the same but not quite - what's the difference?

Reply:
One is part of public API, the other is internal.